packages/select/src/option.vue

```diff
@@ -33,6 +33,7 @@ export default defineComponent({
   props: {
     value: {
       required: true,
+      type: null,
```

Comment:
because it should it be string/number/object we are setting null ? Vue does't support array of types ?
like : 
```suggestion
      type:[String, number,  object],
```